src/vision.cpp

```diff
@@ -29,29 +41,35 @@ Vision::Vision(ros::NodeHandle nh_camera, ros::NodeHandle nh_private)
 
 Vision::~Vision()
 {
+  quit();
+}
+
+rclcpp::node_interfaces::NodeBaseInterface::SharedPtr Vision::getNodeBaseInterface()
+{
+  return node_->get_node_base_interface();
 }
 
 bool Vision::configure()
 {
   std::string streamConfig_rosparam = "";
```

Comment:
I don't really see what the point of creating a variable assigned to an empty string is...seems more straightforward to just put `""` where this variable is used, unless we plan on assigning it to something in the future